synapse_rock/rockcraft.yaml

```diff
@@ -154,6 +154,10 @@ parts:
             # fix issue while creating file
             # https://github.com/element-hq/synapse/issues/17882
             pip3 install --break-system-packages --prefix="/install" --force-reinstall -v "Twisted==24.7.0"
+            # fix issue while upload media to s3
+            # boto3 is installed by synapse-s3-storage-provider
+            # https://github.com/boto/boto3/issues/4398
+            pip3 install --break-system-packages --prefix="/install" --force-reinstall -v "boto3==1.35.99"
```

Comment:
The link says version 1.35.95. Does this one also do the trick?

Reply:
Yes, I tried 99 on staging based on this comment
https://github.com/boto/boto3/issues/4398#issuecomment-2602305618